apps/api/src/domain/games/memory-match/queries/__snapshots__/memory-match-queries.fetch-by-id.e2e.spec.ts.snap

```diff
@@ -1,12 +1,211 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`when querying for a memory match round: fetch by Id when the user is authenticated as a viewer when there is a memory match round with the given ID when the round is public should return the expected response 1`] = `
+{
+  "cardbackImageUrl": "http://localhost/:4600/null/resources/mediaItems/download/9b1deb4d-3b7d-4bad-9bdd-2b0d7b100022",
+  "cards": [
+    {
+      "audioUrl": "http://localhost/:4600/null/resources/mediaItems/download/9b1deb4d-3b7d-4bad-9bdd-2b0d7b100201",
+      "imageUrl": "http://localhost/:4600/null/resources/mediaItems/download/9b1deb4d-3b7d-4bad-9bdd-2b0d7b100101",
+      "sequenceNumber": 1,
+      "text": {
+        "items": [
+          {
+            "languageCode": "en",
+            "role": "original",
+            "text": "Card #1",
+          },
+        ],
+      },
+    },
+    {
+      "audioUrl": "http://localhost/:4600/null/resources/mediaItems/download/9b1deb4d-3b7d-4bad-9bdd-2b0d7b100202",
```

Comment:
You need to inject the `GLOBAL_PREFIX` into the config. I'm also wondering why there's a slash after localhost. I hope we don't have this in other tests. Let's fix this.

Reply:
trailing slash was an error in my env file